src/Components/StepTwo/StepTwo.js

```diff
@@ -1,63 +1,284 @@
+                        value={inputs.email} 
+                        onChange={handleChange}/>
+                   {errors.email && <div>{errors.email?.email}</div>}
                 </div>
 
                 <div className="homeAddressInput">
-                <div className="addressLine">
+                    <div className="addressLine">
                     <p><span>*</span> Address Line 1</p>
-                    <input type="text" placeholder="Address"/>
+                    <Input 
+                        type={"text"} 
+                        placeholder={"Address"}
+                        name={"addressLine1"} 
+                        value={inputs.address.addressLine1} 
+                        onChange={handleChange}/>
+                    </div>
+                    <div className="addressLine">
+                    <p>Address Line 2</p>
+                    <Input 
+                        type={"text"} 
+                        placeholder={"Address"}
+                        name={"addressLine2"} 
+                        value={inputs.address.addressLine2} 
+                        onChange={handleChange}/>
+                    </div>
                 </div>
-                <div className="addressLine">
-                    <p><span>*</span> Address Line 2</p>
-                    <input type="text" placeholder="Address"/>
-                </div>
-            </div>
 
-            <div className="homeCityInput">
-                <div className="city">
-                    <p><span>*</span> City/Town</p>
-                    <input type="text" name="" id="" placeholder="City"/>
-                </div>
-                <div className="state">
+                <div className="homeCityInput">
+                    <div className="state">
                     <p><span>*</span> State</p>
-                    <input type="text" name="" id="" placeholder="State"/>
-                </div>
-                <div className="zip">
+                    <Input 
+                        type={"text"} 
+                        placeholder={"State"}
+                        name={"state"} 
+                        list={"state"} 
+                        value={inputs.address.state} 
+                        onChange={handleChangeAddressState}/>
+                    <datalist id={"state"}>
+                    {dataFetch.map((state, index) => (
+                    <option value={state.name || state.code} 
+                    key={`${state.name}`+ index} />
+                      ))}
+                    </datalist>
+                    </div>
+                    <div className="city">
+                    <p><span>*</span> City/Town</p>
+                    <Input 
+                        type={"text"} 
+                        placeholder={"City"}
+                        name={"city"} 
+                        list={"city"} 
+                        disabled={inputs.address.state === "" ? true : false} 
```

Comment:
I think this can be shortened, since inputs.address.state === "" already returns boolean value.